packages/connect-conformance/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@connectrpc/connect-conformance",
   "version": "1.4.0",
   "private": true,
+  "type": "module",
```

Comment:
Switching this to `module` allows us to build ESM artifacts on Node16 module resolution. But, it raises a question of whether we even need to build both here. Should we maybe only build one? And if so, should it be `commonjs` like `protoc-gen-connect-es` and `connect-migrate`?

Reply:
The package is used by both ESM and CommonJS, right? In that case, let's use the same pattern we use elsewhere.

Reply:
`connect-cloudflare` was the only one that was CJS. But I just updated it to be `type: module` so we could probably remove the CJS build here?

Reply:
protoc-gen-connect-es and connect-migrate are just binaries, they don't export code, but connect-conformance does. Let's use the same pattern everywhere where we export code.

Reply:
Works for me.